docker-compose.yml

```diff
@@ -579,6 +582,120 @@ services:
     - kafka-broker
     - kafka-zookeeper
 
+  IntegrationTests2:
```

Comment:
`IntegrationTests2`? Consider renaming or at least adding a comment to explain why this one is just "2".

Reply:
Oh, I see in the PR description:
>Added new docker-compose targets for now, to avoid breaking existing pipelines. The old ones can be cleaned up when we remove the old pipelines

Reply:
> I could rename it to `NukeIntegrationTests` for now 

If it's not too much trouble, thanks!